libs/zard/src/lib/components/input/input.directive.ts

```diff
@@ -0,0 +1,30 @@
+import { ClassValue } from 'class-variance-authority/dist/types';
+
+import { Directive, ElementRef, inject, input, OnInit, Renderer2 } from '@angular/core';
+
+import { mergeClasses, transform } from '../../shared/utils/utils';
+import { inputVariants, ZardInputVariants } from './input.variants';
+
+@Directive({
+  selector: 'input[z-input]',
```

Comment:
Adding textarea here, we don't need another directive for that.

```suggestion
  selector: 'input[z-input], textarea[z-input]',
```

Reply:
Resolved!